storage/postgresql/schema/storage.sql

```diff
@@ -0,0 +1,224 @@
+-- PostgreSQL version of the tree schema.
+--
+-- Each statement must end with a semicolon, and there must be a blank line before the next statement.
+-- This will ensure that the testdbpgx tokenizer will handle semicolons in the PL/pgSQL functions correctly.
+
+-- ---------------------------------------------
+-- Tree stuff here
+-- ---------------------------------------------
+
+-- Tree parameters should not be changed after creation. Doing so can
+-- render the data in the tree unusable or inconsistent.
+CREATE TYPE TreeState AS ENUM ('ACTIVE', 'FROZEN', 'DRAINING');
+
+CREATE TYPE TreeType AS ENUM ('LOG', 'MAP', 'PREORDERED_LOG');
+
+
+CREATE TABLE IF NOT EXISTS Trees(
+  TreeId                BIGINT NOT NULL,
+  TreeState             TreeState NOT NULL,
+  TreeType              TreeType NOT NULL,
+  DisplayName           VARCHAR(20),
+  Description           VARCHAR(200),
+  CreateTimeMillis      BIGINT NOT NULL,
+  UpdateTimeMillis      BIGINT NOT NULL,
+  MaxRootDurationMillis BIGINT NOT NULL,
+  Deleted               BOOLEAN,
+  DeleteTimeMillis      BIGINT,
+  PRIMARY KEY(TreeId)
+);
+
+-- This table contains tree parameters that can be changed at runtime such as for
+-- administrative purposes.
+CREATE TABLE IF NOT EXISTS TreeControl(
+  TreeId                  BIGINT NOT NULL,
+  SigningEnabled          BOOLEAN NOT NULL,
+  SequencingEnabled       BOOLEAN NOT NULL,
+  SequenceIntervalSeconds INTEGER NOT NULL,
+  PRIMARY KEY(TreeId),
+  FOREIGN KEY(TreeId) REFERENCES Trees(TreeId) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS Subtree(
+  TreeId               BIGINT NOT NULL,
+  SubtreeId            BYTEA NOT NULL,
+  Nodes                BYTEA NOT NULL,
+  SubtreeRevision      INTEGER NOT NULL,
```

Comment:
Pretty sure this is not used now.

Reply:
Addressed by https://github.com/google/trillian/pull/3644/commits/8102b6ade7aa508dd2ae255dd5e721b74f522541.